impl/src/main/java/com/okta/sdk/impl/client/DefaultClientBuilder.java

```diff
@@ -329,8 +392,55 @@ public ClientBuilder setOrgUrl(String baseUrl) {
         return this;
     }
 
+    @Override
+    public ClientBuilder setAuthorizationMode(String authorizationMode) {
+        // if 'authorizationMode' is ever supplied, it should only be "PrivateKey" (OAUTH2 flow);
+        // else do not set it as it defaults to SSWS flow.
+        if (authorizationMode != null) {
+            if (authorizationMode.equals("PrivateKey")) {
```

Comment:
The `Assert` class should provide logic to collapse this if/else into a single line